packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -47,7 +47,7 @@ export const FinalFormSelect = <T,>({
     const endAdornment = !required ? (
         <ClearInputAdornment
             position="end"
-            hasClearableContent={Boolean(multiple ? (Array.isArray(value) ? value.length : value) : value)}
+            hasClearableContent={Boolean(multiple ? (Array.isArray(value) ? value.length : value) : value === 0 ? `${value}` : value)}
```

Comment:
```suggestion
            hasClearableContent={Boolean(multiple ? (Array.isArray(value) ? value.length : value !== undefined) : value !== undefined)}
```

IMO checking for undefined is better here. @jamesricky do you think this could lead to problems?

Reply:
I'd also prefer checking for `undefined`. 
Theoretically we could also have `false` as a valid value in the future, then that would work as well.